File: course 1/Informatics/lab1/lab1_code.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include <locale.h>

void finder()
{
    float x, y;
    printf("Введите первое число:");
    scanf("%f", &x);

    printf("Введите второе число:");
    scanf("%f", &y);

    float res1 = (pow(x, 3) + pow(y, 3))/2;
    float res2 = sqrt(abs(x) * abs(y));

    printf("Среднее арифметическое кубов:%f\n", res1);
    printf("Среднее арифметическое модулей:%f\n", res2);
    
    int ans;
    printf("Вернуться в меню?(1/0):");
    scanf("%i", &ans);

    if (ans == 1)
    {
        finder();
    }
}

int main()
{
    setlocale(LC_ALL, "");
    
    finder();

    return 0;
}

File: course 1/Enter the programming/Лаб 1/lab1_code.py
from math import *

print("Введите значения x, y и z (каждое в отдельной строке)")
x, y, z  = float(input()), float(input()), float(input())

# s = y**(sqrt(abs(x))**3) + cos(y)**3 * abs(x - y) * (1 + sin(x)**2) / exp(abs(x-y))
s = y**(abs(x)**(1/3)) + cos(y)**3 * abs(x - y)\
      * (1 + sin(z)**2/sqrt(x + y)) / (exp(abs(x-y)) + x/2)

print("s = ", s)